src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -10,8 +10,10 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic.SyntaxFacts
 Imports Roslyn.Test.Utilities
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module ParserTestUtilities
+    Friend ReadOnly Property PooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool(If(Environment.Is64BitProcess, 96, 64))
```

Comment:
@AlekseyTs  Just experimenting see if made any difference, on balance 64 is ok for both. So I'll revert to 64.